taccsite_cms/_settings/form_plugin.py

```diff
@@ -0,0 +1,128 @@
+"""Configure djangocms_forms and related apps"""
+
+from django.utils.translation import gettext_lazy as _
+
+########################
+# KNOWN ISSUES
+########################
+
+# Unable to export to Excel nor YAML
+# (No success hiding those formats from editor)
+# https://github.com/avryhof/djangocms-forms/issues/8
+
+########################
+# DJANGO_RECAPTCHA
+# https://github.com/avryhof/django-recaptcha/
+########################
+
+# To properly avoid client error about using test keys
+# RECAPTCHA_PUBLIC_KEY = '__this_must_be_on_server_not_here__'
+# RECAPTCHA_SECRET_KEY = '__this_must_be_on_server_not_here__'
+
+# To silence server error about using test keys
+# SILENCED_SYSTEM_CHECKS = ['captcha.recaptcha_test_key_error']
+
+########################
+# DJANGOCMS_FORMS
+# https://github.com/avryhof/djangocms-forms/
+########################
+
+# DJANGOCMS_FORMS_PLUGIN_MODULE = ('Generic')
```

Comment:
Are the commented settings important to keep around?

Reply:
No. I will remove some and simplify captcha notes.

Reply:
062ba78